test/pbbuilder_test.rb

```diff
@@ -25,11 +25,38 @@ class PbbuilderTest < ActiveSupport::TestCase
 
     assert_equal "Hello world", person.name
     assert_equal "Friend #1", person.friends.first.name
-    assert_equal ["ok", "that's", "cool"], person.field_mask.paths
+    assert_equal ["cool"], person.field_mask.paths
     assert_equal "Manuelo", person.best_friend.name
     assert_equal "Eggs", person.favourite_foods["Breakfast"]
   end
 
+  test "duplicated enum fields don't happen" do
```

Comment:
nit: "don't happen" - maybe the same? "sets the last value of the repeated field to be the only value"?